src/hotspot/os_cpu/linux_ppc/os_linux_ppc.cpp

```diff
@@ -277,10 +278,10 @@ JVM_handle_linux_signal(int sig,
 
   JavaThread* thread = NULL;
   VMThread* vmthread = NULL;
-  if (os::Linux::signal_handlers_are_installed) {
+  if (PosixSignals::are_signal_handlers_installed()) {
     if (t != NULL) {
       if(t->is_Java_thread()) {
-        thread = t->as_Java_thread();
+        thread = (JavaThread*)t;
```

Comment:
Mis-merge? Please put this back to t->as_Java_thread()

Reply:
Merge issue, fixed.